python/tvm/_ffi/runtime_ctypes.py

```diff
@@ -262,9 +262,23 @@ def max_thread_dimensions(self):
         """
         return json.loads(self._GetDeviceAttr(self.device_type, self.device_id, 8))
 
-    def sync(self):
+    def create_stream(self):
+        """Create a new runtime stream at the context."""
```

Comment:
document the return value per numpy docs convention(see documentation for max_thread_dimensions)

Reply:
Given that the stream we have is quite low-level (not managed by python) we might want to think about how to introduce a better pythonic wrapper later. As a result, we might want to name the API more carefully to indicate it is a low-level api.

How about we do `create_raw_stream`, and add a note that the user expects to free the stream after use. Do the same thing for the other apis